util/metadata.go

```diff
@@ -1,42 +1,79 @@
 package util
 
 import (
+	"archive/tar"
+	"bytes"
+	"compress/gzip"
 	"fmt"
+	"io"
 	"io/ioutil"
-	"os"
-	"path/filepath"
+	"net/http"
 	"strings"
 )
 
+const url = "https://api.github.com/repos/web-platform-tests/wpt-metadata/tarball"
+
 // CollectMetadata iterates through wpt-metadata repository and returns a
 // map that maps a test path to its META.yml file content.
-func CollectMetadata() map[string][]byte {
-	ymlFiles := []string{}
-	curPath, err := os.Executable()
+func CollectMetadata() (res map[string][]byte, err error) {
+
+	resp, err := http.Get(url)
```

Comment:
That makes sense. But one question I have here is - since this code is located at wpt-metadata repository, which is used as a third-party library to wpt.fyi, does it mean all third party libraries have to conform to this practice (using urlfetch rather than http.Get)?